components/expanding-components/section.scss

```diff
@@ -0,0 +1,52 @@
+.msds-expanding-section {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  // padding-top: 96px;
+  // padding-bottom: 80px;
+
+  &--closed {
+    display: flex;
+    transition: all 0.15s ease-in-out;
+  }
+
+  &--open {
+    display: none;
+    transition: all 0.15s ease-in-out;
+  }
+
+  &__content {
+    max-height: 0;
+    overflow: hidden;
+    transition: all 0.25s ease-in-out;
+  }
+
+  .msds-btn-floating {
+    padding-bottom: 42px;
+  }
+
+  & input[type='checkbox'],
+  & input[type='radio'] {
+    display: none;
+
+    &:checked {
+      ~ .msds-expanding-section__buttons-container {
+        background: red;
+        height: 100px;
+        width: 100%;
+
+        .msds-expanding-section--closed {
+          display: none;
+        }
+
+        .msds-expanding-section--open {
```

Comment:
i does not make sense to me why you have to specify display flex here aswell as it already has been declared on the block on line 2

Reply:
its the same situation, I am toggling display none/flex when users click the button